integrations/standalone/tests/integration/project/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>ch.ivyteam.variable</groupId>
+  <artifactId>integration</artifactId>
+  <version>11.4.0-SNAPSHOT</version>
+  <packaging>iar-integration-test</packaging>
+
+  <properties>
+    <engine.page.url>https://product.ivyteam.io/</engine.page.url>
+    <project.build.plugin.version>11.4.0-SNAPSHOT</project.build.plugin.version>
```

Comment:
as we now have an ivy project in this repo, we should also adjust our raise version scripts:
- add raise-build-plugin-version.sh
- add this repo to the raise project build plugin version job
- adjust the raise-version.sh script to also update this project version